wiki/Contests/KPMC/1/en.md

```diff
@@ -0,0 +1,86 @@
+| Results announcement | 2025-08-28 at 12 UTC |
+
+The playtester's pick and top 5 beatmaps were showcased during the [results announcement YouTube premiere](https://www.youtube.com/watch?v=77D7EWC5irQ).
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ::{ flag=ID }:: [Ilham](https://osu.ppy.sh/users/3057154), ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699) |
+| Designer | ::{ flag=ID }:: [kim hyunjin](https://osu.ppy.sh/users/9456733) |
+| Playtester | ::{ flag=ID }:: [_yea](https://osu.ppy.sh/users/20834055) |
+| Judge | ::{ flag=PH }:: [Hytex](https://osu.ppy.sh/users/8536263), ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785), ::{ flag=US }:: [Monoseul](https://osu.ppy.sh/users/16010604), ::{ flag=KR }:: [secXcscX](https://osu.ppy.sh/users/13543418) |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months osu!supporter, Profile badge (TBD), Beatmap banner, Ranked status[^ranked] |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months osu!supporter, Beatmap banner, Ranked status[^ranked] |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months osu!supporter, Beatmap banner, Ranked status[^ranked] |
+| Playtester's Pick[^playtester] | 1 month osu! supporter |
+
+![KPMC#1 badge](img/badge.jpg)
+
+## Links
+
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/2097919)
+- [Results sheet](https://docs.google.com/spreadsheets/d/1ZhCLU6uAazuf4z1JF8gp_Wdn3h0oKDWZw5ujzCcqo60/edit?usp=sharing)
+- [Entries](https://drive.google.com/drive/folders/1wDf2wYEbmhQQ4CpQ53BB_txBLbuEVDzv)
+
+## Results
+
+More detailed scores and judge comments are available to view on the [Mappers' Guild website](https://mappersguild.com/contests/results?contest=685f8a94a9d485e4a026997e).
+
+| Place | Mapper | Song (map link) | Raw score (160) | Standardised score |
+| :-: | :-- | :-- | :-- | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=ID }:: [eZmmR](https://osu.ppy.sh/users/8647138) | [2NE1 - COME BACK HOME](https://osu.ppy.sh/beatmapsets/2441893#mania/5327786) | 125 | 2.88 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=CA }:: [Livium](https://osu.ppy.sh/users/9828552) | [ILLIT - jellyous](https://osu.ppy.sh/beatmapsets/2442615#mania/5329664) | 127 | 2.81 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=US }:: [homwu](https://osu.ppy.sh/users/18292810) | [ENHYPEN - Polaroid Love](https://osu.ppy.sh/beatmapsets/2442060#mania/5330875) | 124 | 2.53 |
+| Playtester's Pick[^playtester] | ::{ flag=KR }:: [Turrim](https://osu.ppy.sh/users/3846265) | [TAK - DKBK feat. Tenko Shibuki](https://osu.ppy.sh/beatmapsets/2442025#mania/5328142) | 110 | -0.22 |
+
+## Ruleset
+
+### Contest rules
+
+- Submissions must be in `.osz` format.
+- Submissions must be made for the osu!mania game mode only and between 4K and 7K.
```

Comment:
The current sentence reads somewhat run-on with the inclusion of the and. Best to either use a full stop and a connector, or even better have them as separate rules. Keeps it cleaner.

```suggestion
- Submissions must be made for the osu!mania game mode only.
- Submissions must be between 4K and 7K.
```